src/ConfigFactory.Avalonia/Builders/DropdownControlBuilder.cs

```diff
@@ -0,0 +1,43 @@
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Data;
+using Avalonia.Layout;
+using ConfigFactory.Core;
+using ConfigFactory.Core.Attributes;
+using System.Reflection;
+
+namespace ConfigFactory.Avalonia.Helpers;
+
+internal static class DropdownBuilder
+{
+    internal static ComboBox Build(IConfigModule context, PropertyInfo propertyInfo, DropdownConfigAttribute dca)
+    {
+        var combobox = new ComboBox
+        {
+            DataContext = context,
+            HorizontalAlignment = HorizontalAlignment.Stretch,
+            ItemsSource = dca.GetItemsSource(context),
+            VerticalAlignment = VerticalAlignment.Top,
+        };
+
+        if (!string.IsNullOrWhiteSpace(dca.DisplayMemberPath))
+        {
+            // e.g. KeyValuePair.Key
```

Comment:
This comment is kind of unless and/or confusing. It's explaining how to use the `DropdownControlBuilder` in a place where no user would typically look. Would be better in code documentation on the attribute property.

```suggestion
```